lib/processors/resourceListCreator.js

```diff
@@ -123,15 +123,14 @@ function makeResourcesJSON(list, prefix) {
 /**
  * Whether the detection of orphans should result in a build failure.
```

Comment:
How does this JSDoc comment match the following export? Looks as if this comment stems from a time when there was a separate property `failOnOrphans` (maybe shortly after the migration from the Java code)

Reply:
Didn't notice that. I've updated the JSDoc. Should be checked again when we make the processor public.